agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -704,3 +704,67 @@ func TestTaskMetadataValidator(t *testing.T) {
 
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+// TestExecutionRole verifies that task can use the execution credentials to pull from ECR and
+// send logs to cloudwatch with awslogs driver
+func TestExecutionRole(t *testing.T) {
+	if os.Getenv("TEST_EXECUTION_ROLE") != "true" {
+		t.Skip("TEST_EXECUTION_ROLE was not set to true")
+	}
+
+	RequireDockerVersion(t, ">=17.06.2-ce") // awslogs drivers with execution role available from docker 17.06.2
```

Comment:
Is there a remote API version which supports execution role, that we can use to compare, instead of the Docker version itself?

Reply:
It's the same as version in this case, since this version has a patch so it will have the same API version as the one without patch.